### ISSUE TITLE
Modifying SPA actions throws error

### ISSUE THREAD

**Issue by pabogdan** [nengo/nengo_gui#609]
Modifying actions throws error instead of running simulation with new actions (when manually restarting the server the described actions are correct and function as expected)
```bash
Traceback (most recent call last):
  File "/home/bogdanp/nengo_gui/nengo_gui/server.py", line 142, in ws_viz_component
    component.update_client(client)
  File "/home/bogdanp/nengo_gui/nengo_gui/components/netgraph.py", line 373, in update_client
    self.check_for_reload()
  File "/home/bogdanp/nengo_gui/nengo_gui/components/netgraph.py", line 63, in check_for_reload
    self.reload()
  File "/home/bogdanp/nengo_gui/nengo_gui/components/netgraph.py", line 86, in reload
    self._reload(code=code)
  File "/home/bogdanp/nengo_gui/nengo_gui/components/netgraph.py", line 210, in _reload
    old_obj = eval(safe_eval % item, old_locals)
  File "<string>", line 1
    (lambda *a, **b: list(a) + b.values())(model.<Buffer "cortex">)[0]
                                                 ^                                                                        
SyntaxError: invalid syntax
```

**Comment by Seanny123** [comment]
Can you paste the code you modified, so that I can reproduce the error locally?

**Comment by Seanny123** [comment]
Ah. I just replicated it. Literally any modification throws an error.

**Comment by pblouw** [comment]
I'm now unable to replicate this bug on OSX or Linux.

**Comment by pabogdan** [comment]
@pblouw I think this has been fixed by #614
